### ISSUE TITLE
Motion Sickness and A11y Accessibility Violation

### ISSUE THREAD

**Issue by ujjwalvivek** [ujjwalvivek/portfolio#2]
### **Problem:** 
Too much animation, no prefers-reduced-motion support

---

### **Fix**:

- [x] Respect `prefers-reduced-motion`
- [x] Add a "Low Chaos Mode" toggle for all motion-disabled experience
- [x] Motion/animation reduced version
- [x] For sensitive users or poor hardware
